frontend/src/components/pages/Scheduling/VolunteerInformation.tsx

```diff
@@ -13,13 +25,105 @@ const VolunteerInformation = ({
 }: SchedulingStepProps) => {
   const { previous, next } = navigation;
   const history = useHistory();
-  //   const { insert form fields for this page here } = formData;
+  const {
+    startTime,
+    endTime,
+    volunteerNeeded,
+    volunteerTime,
+    pickupLocation,
+    isPickup,
+    notes,
+  } = formValues;
+
+  const handleChange = (e: boolean | string, name: string) => {
+    setForm({ target: { name, value: e } });
+  };
+
+  const volunteerNeededValues = ["Yes", "No"];
+  const volunteerAssistanceValues = [
+    "Pickup (food rescue)",
+    "Unloading (on site)",
+  ];
+  const volunteerRequiredHelperText =
+    "A volunteer is a community fridge member who will assist with donation dropoffs. Information of the volunteer assigned will be provided.";
 
   return (
-    // Insert Volunteer Information page here
-    <Container>
+    <Container p="30px">
       <SchedulingProgressBar activeStep={2} totalSteps={4} />
-      <Text textStyle="mobileHeader2">Volunteer Information Page</Text>
+      <Text textStyle="mobileHeader2" mt="2em">
+        Volunteer Information
+      </Text>
+      <Box
+        p="1.5em"
+        bg="squash.100"
+        align="left"
+        m="2em 0 3em"
+        borderWidth="1px"
+        borderColor="#6C6C84"
+        borderRadius="5px"
+      >
+        <Text textStyle="mobileHeader4">Proposed dropoff time</Text>
+        <Text textStyle="mobileBody">{new Date(startTime).toDateString()}</Text>
+        <Text textStyle="mobileBody">
+          {new Date(startTime).toLocaleTimeString()}-
+          {new Date(endTime).toLocaleTimeString()}
+        </Text>
+        <Text textStyle="mobileBody">One time donation (hard-coded)</Text>
+      </Box>
+      <RadioSelectGroup
+        name="volunteer-required"
+        label="Do you require a volunteer?"
+        value={volunteerNeeded ? "Yes" : "No"}
+        values={volunteerNeededValues}
+        isRequired
+        helperText={volunteerRequiredHelperText}
+        onChange={(e: string) => {
+          handleChange(e === "Yes", "volunteerNeeded");
+        }}
+      />
+      <br />
```

Comment:
Should we add padding to the RadioSelectGroup instead of <br> tags?

Reply:
hmm yeah I can update RadioSelectGroup to take margin as a prop?

Reply:
just hardcoded a top/bottom margin in the RadioSelectGroup component, should be fine since all the pages that use the component should have similar styling